### DISCUSSION TITLE
Ideas - Assessments - 20 Questions

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#102] [Ideas]
# Requirement

20 MCQs in the format given in the attached spreadsheet for different technical topics.

# Topics

- Spring Framework
- Spring Microservices
- AWS
- MySQL

# Sample Document

[Sample Assessment.csv](https://github.com/akash-coded/spring-framework/files/8795332/Sample.Assessment.csv)

# To be Uploaded

Excel sheet(s) with MCQ questions for the given topics

**Comment by Svishva** [comment]
MySQL

[Sample.Assessment MYSQL.xlsx](https://github.com/akash-coded/spring-framework/files/8795560/Sample.Assessment.MYSQL.xlsx)


AWS 

[Sample.Assessment AWS.csv](https://github.com/akash-coded/spring-framework/files/8795626/Sample.Assessment.AWS.csv)

**Comment by KalaiselviRajavadivel** [comment]
# Spring and spring boot
[Sample.Assessment.csv](https://github.com/akash-coded/spring-framework/files/8795600/Sample.Assessment.csv)

**Comment by Thiripurasundari-1223** [comment]
# Spring boot and microservices

[Assessment-springboot&microservices.csv](https://github.com/akash-coded/spring-framework/files/8795604/Assessment-springboot.microservices.csv)

**Comment by kunalki** [comment]
Microservices.csv

**Comment by Narutoram** [comment]
[Sample.Assessment_mysql nd aws.csv](https://github.com/akash-coded/spring-framework/files/8795708/Sample.Assessment_mysql.nd.aws.csv)

**Comment by kunalki** [comment]
[Spring mcq (1).csv](https://github.com/akash-coded/spring-framework/files/8796141/Spring.mcq.1.csv)
